src/luarocks/fetch/cvs.lua

```diff
@@ -20,9 +20,14 @@ function cvs.get_sources(rockspec, extract, dest_dir)
    assert(type(rockspec) == "table")
    assert(type(dest_dir) == "string" or not dest_dir)
 
+   local cvs_cmd = rockspec.variables.CVS
+   if not fs.execute_quiet(cvs_cmd, "--version") then
+      return nil, "'"..cvs_cmd.."' program not found. Is CVS installed? You may want to edit variables.CVS"
```

Comment:
> It doesn't have to be in PATH if it's an absolute path.

Sure, I was just catering for the common case. If git is not found, most likely it is simply not installed and installing via the OS's usual methods will put it there.